.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint Code Base
+
+on:
+  push:
+    branches: [ staging, production ]
+  pull_request:
+    branches: [ staging, production ]
+
+jobs:
+  lint:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v5
```

Comment:
Not sure about this -- can we use a local linter instead, then it can be run in dev as well

Reply:
Sure, do you have a suggestion?